docs/cadence_migration_guide/index.md

```diff
@@ -6,19 +6,60 @@ sidebar_label: Cadence 1.0 Migration Guide
 
 # Cadence 1.0 Migration Guide
 
-Coming in 2024, the network will be upgrading to the Cadence 1.0. This means that all applications will need to prepare and migrate their existing Cadence smart contracts, scripts, and transactions for the update. If you do not update your code, your applications will become non-functional after the network upgrade. The specific upgrade date will be announced later.
+Coming in 2024, the network will be upgrading to [Cadence 1.0](https://flow.com/upgrade/cadence-1). This means that all applications will need to prepare and migrate their existing Cadence smart contracts, scripts, and transactions for compatibility with the update. If you do not update your contracts, your applications will become non-functional after the network upgrade. Read more about the roadmap [here](https://flow.com/upgrade/cadence-1).
 
-We have created several resources to help developers update their applications. Use the following links to start upgrading your contracts and to understand how the network upgrade will affect your application:
+### Upgrading Contracts
 
-- [Understand the upgrade, the timeline, and how it affects you](https://forum.flow.com/t/cadence-1-0-upgrade-plan/5477)
-- [Learn how Cadence will be changing for 1.0](https://forum.flow.com/t/update-on-cadence-1-0/5197)
-- [Learn how to update your Flow CLI to test against Cadence 1.0](https://forum.flow.com/t/update-on-cadence-1-0/5197/7)
-- [Learn how to update your Non-Fungible Token contracts for Cadence 1.0](./nft-guide.mdx)
-- [Learn how to update your Fungible Token contracts for Cadence 1.0](./ft-guide.mdx)
-- [Check out the Cadence language 1.0 docs](https://cadencelang.dev/docs/1.0/)
+To ensure your contracts are fully operational with Cadence 1.0, follow these essential steps:
+
+1. **Understand the Changes:** Start by familiarizing yourself with the [changes](https://forum.flow.com/t/update-on-cadence-1-0/5197) to identify how they might affect your code.
+2. **Modify Your Code:** Update your contracts, transactions, and scripts in accordance with the new changes.
+3. **Test Your Code:** Use the [Cadence 1.0 Emulator](https://www.loom.com/share/4467610b7beb4ebbaabed6b430dc25c4?sid=14ecb3e7-e933-409c-a6a4-add40c6971d0) to test your code by running:
```

Comment:
I don't know if linking the video for "Cadence 1.0 Emulator" makes sense here. probably would make sense to link to the release and then explain that it includes all the new features and updated standards and core contracts. Then we can say something like, "Check out a video from the Flow team about how to install and use the Cadence 1.0 emulator."

Reply:
done